app/win/BUILD.gn

```diff
@@ -37,7 +37,7 @@ copy("dist_resources") {
   ]
 
   public_deps = [
-    "//chrome_elf",
+    "//chrome/chrome_elf",
```

Comment:
why a public dep for this?

Reply:
@bbondy?

Reply:
oh, sorry, it was already here so don't worry about it